packages/block-editor/src/components/media-replace-flow/test/index.js

```diff
@@ -78,6 +101,16 @@ describe( 'General media replace flow', () => {
 			} )
 		);
 
+		/**
+		 * The popover will be displayed and positioned with a slight delay,
+		 * so we're opting to wait until that happens.
+		 */
+		await waitFor( () =>
+			expect(
+				getWrappingPopoverElement( screen.getByRole( 'menu' ) )
+			).toHaveStyle( { top: '13px', left: '0' } )
+		);
```

Comment:
Again, to tell a better story, I wouldn't look for the `.getByRole( 'menu' )` element. The test is not interested in it. Use the `link` element instead:
```js
const link = screen.getByRole( 'link' ); // or await findByRole if it doesn't appear sync
await waitFor( () => expect( getWrappingPopover( link ) ).toHaveStyle() );
expect( link ).toHaveAttribute();
```

Reply:
Sure. Not a huge deal I guess, but happy to make things more clear. Done in 85d318d80ea60f9a2caaad6c179a40f77f7b10da